src/components/Editor/Tab.js

```diff
@@ -192,7 +192,9 @@ class Tab extends PureComponent<Props> {
           source={source}
           shouldHide={icon => ["file", "javascript"].includes(icon)}
         />
-        <div className="filename">{getTruncatedFileName(source)}</div>
+        <div className="filename">
+          {getRawSourceURL(getTruncatedFileName(source))}
```

Comment:
maybe we can have `getTruncatedFileName` do `getRawSourceURL` because I assume we always want to do this.

It'd be nice to have a unit test for `getTruncatedFileName` that demonstrates the behavior.
Or have a unit test for Tab.js that demonstrates it

Reply:
Good shout.